### PR TITLE
Update error msg when adding Exec or Admin

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#16293

*Description of changes:*
- Update to error message for adding a person or business for WILL, AFFE and LETA transfers

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by dimak1** [comment]
Opened PR as a Draft for now, as I need to confirm error message with the Designers.

File: ppr-ui/src/resources/mhr-transfers/transfers-errors.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { ApiTransferTypes } from '@/enums'
 
+/* eslint-disable max-len */
```
Comment (dimak1) [inline ppr-ui/src/resources/mhr-transfers/transfers-errors.ts:3]: So that we don't have to break long messages into multiple lines

**Review by doug-lovett — APPROVED** [review]

**Review by orelbn — APPROVED** [review]

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1348-ish3gbx1.web.app

**Review by cameron-eyds — APPROVED** [review]

**Pull request merged**